src/styles/main.scss

```diff
@@ -0,0 +1,62 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
```

Comment:
- You don't need to specify font for each element, just to html
- And the same with margins/paddings, reset them for tags that actually need it, as div/span/some other tags don't have any of them